spec/jobs/evss/disability_compensation_form/submit_form526_spec.rb

```diff
@@ -26,6 +26,7 @@
 
     context 'when the base class is used' do
       it 'raises an error as a subclass should be used to perform the job' do
+        allow_any_instance_of(Form526Submission).to receive(:prepare_for_evss!).and_return(nil)
```

Comment:
Why was this added?

Reply:
This (and wherever else I've added this stub) is an integration test for EVSS submission which cares only about the EVSS segment, but includes our RRD behavior in the test surface area. Since `prepare_for_evss!` is only getting more complicated, it's better for this test to just stub it out rather than do all the requisite VCRing.

It's not the best setup but refactoring these tests should be done in a separate PR.